go/tools/go-upgrade/go-upgrade.go

```diff
@@ -432,7 +432,6 @@ func updateBootstrapVersionInCodebase(old, new string, newGoVersion *version.Ver
 		return nil
 	}
 	files, err := getListOfFilesInPaths([]string{
-		"./docker/base",
```

Comment:
The CI workflow failed at midnight (UTC) because of this, `./docker/base` does not exist anymore since #15620. Only need to change the `go-upgrade.go` script on `main`.